framework/Supports/SocialsFeed.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+namespace JustCoded\ThemeFramework\Supports;
+
+use JustCoded\ThemeFramework\Admin\ThemeSettings;
+use Facebook\Facebook as FacebookSDK;
+use MetzWeb\Instagram\Instagram as InstagramSDK;
+use Abraham\TwitterOAuth\TwitterOAuth;
+
+
+/**
+ * Class SocialsFeed
+ *
+ * @package JustCoded\ThemeFramework\Supports
+ */
+class SocialsFeed {
+
+	/**
+	 * Posts array
+	 *
+	 * @var $fb_posts
+	 */
+	public $posts = array();
+
+	/**
+	 * SocialsFeed constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'register_socials_posttype' ) );
+		add_action( 'init', array( $this, 'register_socials_taxonomy' ) );
+		add_action( 'init', array( $this, 'create_settings_panel' ) );
+
+		wp_schedule_event( time(), 'hourly', 'get_social_posts' );
```

Comment:
for wp cron scheduler you should use code like this

add_action('jtf_social_sheduler', array($this, 'get_social_posts'));
if (! wp_next_scheduled ( 'jtf_social_sheduler')) {
    wp_schedule_event(time(), 'hourly', 'jtf_social_sheduler');
}